integreat_cms/cms/templates/organizations/organization_form.html

```diff
@@ -15,9 +18,25 @@ <h1 class="heading">
 				{% trans 'Create new organization' %}
 			{% endif %}
 		</h1>
-		{% if perms.cms.change_organization %}
-			<button class="btn">{% trans 'Save' %}</button>
-		{% endif %}
+		<div class="flex flex-wrap justify-between gap-4"> 
+			{% if perms.cms.change_organization %}
+				<div class="flex flex-wrap gap-4">
+					<button class="btn">{% trans 'Save' %}</button>
+				</div>
+			{% endif %}
+			{% if form.instance.id and perms.cms.delete_organization %}
+				<div class="flex flex-wrap gap-4">
+					<button title="{% trans 'Delete organization' %}" class="btn confirmation-button btn-red"
+							data-confirmation-title="{{ delete_dialog_title }}"
+							data-confirmation-text="{{ delete_dialog_text }}"
+							data-confirmation-subject="{{ form.name.value }}"
+							data-action="{% url 'delete_organization' region_slug=request.region.slug slug=form.instance.slug %}">
+						<i data-feather="trash-2" class="mr-2"></i>
+						{% trans 'Delete this organization' %}
+					</button>
+				</div>
+			{% endif %}
```

Comment:
```suggestion
			{% if form.instance.id and perms.cms.delete_organization %}
				<div class="flex flex-wrap gap-4">
					<button title="{% trans 'Delete organization' %}" class="btn confirmation-button btn-red"
							data-confirmation-title="{{ delete_dialog_title }}"
							data-confirmation-text="{{ delete_dialog_text }}"
							data-confirmation-subject="{{ form.name.value }}"
							data-action="{% url 'delete_organization' region_slug=request.region.slug slug=form.instance.slug %}">
						<i data-feather="trash-2" class="mr-2"></i>
						{% trans 'Delete this organization' %}
					</button>
				</div>
			{% endif %}
			{% if perms.cms.change_organization %}
				<div class="flex flex-wrap gap-4">
					<button class="btn">{% trans 'Save' %}</button>
				</div>
			{% endif %}
```
I'd put the deletion button to the left

Reply:
Alright :D